test/acceptance/table.gspec

```diff
@@ -181,6 +184,10 @@
 		d2l-wide.scroll-wrapper.right-action:
 			inside partly d2l-wide 10px top, 275px left
 			image file dumps/${path}/objects/d2l-wide.scroll-wrapper.right-action.png, error 20px
+			
+	@if ${isPresent("table-with-tspan")}
+		table-with-tspan.tall-row:
+			width > 100 px
```

Comment:
Add an image comparison? It should be enough to add the table to the baseImageCompare group

Reply:
Then you'll need to run `npm run dump` and add the table-with-tspan entries under the `d2l-table` and `d2l-table-rtl` directories

Reply:
Does that just test that the table renders the same way on all browsers?

Reply:
Yes

Reply:
Oh, I see. Thanks